app/static/js/profile.js

```diff
@@ -0,0 +1,244 @@
+import searchUser from './searchUser.js';
+
+function callback(selected) {
+  console.log('Selected student:', selected);
+  $("#searchStudentsInput").val(`${selected.firstName} ${selected.lastName}`);
+  $("#studentEmail").val(selected.email);
+}
+
+$(document).ready(function() {
+  $("#searchStudentsInput").on("input", function() {
+    console.log("Input event triggered");
+    searchUser("searchStudentsInput", callback);
+  });
+
+  $("#searchIcon").click(function (e) {
+    e.preventDefault();
+    let selected = { firstName: $("#searchStudentsInput").val().split(" ")[0], lastName: $("#searchStudentsInput").val().split(" ")[1] };
+    callback(selected);
+  });
+
+  $("input[name='experienceType']").on("change", function() {
+    toggleOtherExperienceTextarea();
+  });
+
+  $("input[name='experienceHoursOver300']").on("change", function() {
+    toggleTextarea();
+  });
+
+  function toggleTextarea() {
+    var yesRadio = document.getElementById('yes300hours');
+    var textareaContainer = document.getElementById('hoursBelow300Container');
+    if (yesRadio.checked) {
+      textareaContainer.style.display = 'none';
+    } else {
+      textareaContainer.style.display = 'block';
+    }
+  }
```

Comment:
We should be using jquery, because that's what we do elsewhere in our javascript.